sphinx-docs/Lateral-Movement-Guide.md

```diff
@@ -0,0 +1,92 @@
+Windows Lateral Movement Guide
+======================
```

Comment:
make heading style consistent (I know we mix and match through a lot of these docs, but we should work towards standardizing it)

replace `========` with

```
# Windows Lateral Movement Guide
```